extraction/utils/extraction_utils.py

```diff
@@ -0,0 +1,11 @@
+"""This script contains helper utilities for extractors"""
+
+import re
+
+
+def has_domain(url, regex):
+    return re.compile(regex).match(url)
+
+
+def get_youtube_video_id(url):
+    return url.split('/')[4]
```

Comment:
If it just used once then no need to move to utils.

Reply:
It can be private method wherever used.

Reply:
Updated!